packages/vx-event/src/typeGuards.ts

```diff
@@ -0,0 +1,27 @@
+export function isElement(elem?: Element | Event): elem is Element {
```

Comment:
should update all of the event types to also accept `React.MouseEvent | React.TouchEvent`